ui/src/application/AppStore.ts

```diff
@@ -35,15 +35,15 @@ export class AppStore extends BaseStore<IApplication> {
     };
 
     @action
-    public update = async (id: number, name: string, description: string): Promise<void> => {
-        await axios.put(`${config.get('url')}application/${id}`, {name, description});
+    public update = async (id: number, name: string, description: string, priorityDefault: number): Promise<void> => {
+        await axios.put(`${config.get('url')}application/${id}`, {name, description, priorityDefault: Number(priorityDefault)});
         await this.refresh();
         this.snack('Application updated');
     };
 
     @action
-    public create = async (name: string, description: string): Promise<void> => {
-        await axios.post(`${config.get('url')}application`, {name, description});
+    public create = async (name: string, description: string, priorityDefault: number): Promise<void> => {
+        await axios.post(`${config.get('url')}application`, {name, description, priorityDefault: Number(priorityDefault)});
```

Comment:
The Number() shouldn't be needed here, as the priorityDefault param is already a number. The parseInt in a previous comment should ensure, that a number is set inside the state of the application dialog.